File: 2010_2011_tp_note/Polya.py
import numpy as np

from Urne import Urne


def Polya(n, r, b):
    R = np.zeros(n + 1)
    R[0] = r
    for k in range(1, n + 1):
        R[k] = Urne(R[k - 1], k - 1, r, b)
    return R

File: 2010_2011_tp_note/Polya.m
function [R] = Polya(n,r,b )
%UNTITLED3 Summary of this function goes here
%   Detailed explanation goes here
R=zeros(n+1,1);
R(1)=r;
for k = 1:n
    R(k+1) = Urne(R(k),k-1,r,b); %Attention aux indices, l'indice de R commence a 1
end
end

